src/components/buttons/aprove-server-form.js

```diff
@@ -49,6 +49,8 @@ module.exports = {
             await interaction.editReply("Servidor Aprovado");
         }
 
+        client.emit('updateGuilds')
```

Comment:
explicacao no primeiro comentario em src/events/updateGuilds.js
```suggestion
        client.emit('updateGuilds', false);
```